icon-ui.js

```diff
@@ -0,0 +1,29 @@
+define(function(require) {
+    /**
+     * Constructor
+     */
+    var Icon = function($el, options) {
+        // Component instance properties.
+        this.$el = $el;
+        this.$use = this.$el.find('use').eq(0);
```

Comment:
If this is only used to retrieve `xlink:href`, should this instance variable just be that? Do you foresee any other use of `this.$use`?